components/ProfilePage/MoreIdeasForm.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function MoreIdeasForm({
+  currentProfile,
+  personId,
+  onUpdateIdeas,
+}) {
+  const [edit, setEdit] = useState(false);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const adaptedIdeas = event.target.ideas.value;
+    onUpdateIdeas(adaptedIdeas, personId);
+    setEdit(!edit);
+  }
+
+  return (
+    <>
+      <StyledInfo>Geschenkideen:</StyledInfo>
+      <StyledNotesWrapper>
+        {edit === true || !currentProfile.ideas ? (
```

Comment:
you theoretically can write only "edit" (equal to edit === true). But that way it's maybe easier to understand

Reply:
Good to know, but I like it when it's very descriptive :)